constants/locale.go

```diff
@@ -2,508 +2,3360 @@ package constants
-li_BE UTF-8
-li_NL UTF-8
-lij_IT UTF-8
-ln_CD UTF-8
-lo_LA UTF-8
-lt_LT ISO-8859-13
-lt_LT.UTF-8 UTF-8
-lv_LV ISO-8859-13
-lv_LV.UTF-8 UTF-8
-lzh_TW UTF-8
-mag_IN UTF-8
-mai_IN UTF-8
-mg_MG ISO-8859-15
-mg_MG.UTF-8 UTF-8
-mhr_RU UTF-8
-mi_NZ ISO-8859-13
-mi_NZ.UTF-8 UTF-8
-mk_MK ISO-8859-5
-mk_MK.UTF-8 UTF-8
-ml_IN UTF-8
-mn_MN UTF-8
-mni_IN UTF-8
-mr_IN UTF-8
-ms_MY ISO-8859-1
-ms_MY.UTF-8 UTF-8
-mt_MT ISO-8859-3
-mt_MT.UTF-8 UTF-8
-my_MM UTF-8
-nan_TW UTF-8
-nan_TW@latin UTF-8
-nb_NO ISO-8859-1
-nb_NO.UTF-8 UTF-8
-nds_DE UTF-8
-nds_NL UTF-8
-ne_NP UTF-8
-nhn_MX UTF-8
-niu_NU UTF-8
-niu_NZ UTF-8
-nl_AW UTF-8
-nl_BE ISO-8859-1
-nl_BE.UTF-8 UTF-8
-nl_BE@euro ISO-8859-15
-nl_NL ISO-8859-1
-nl_NL.UTF-8 UTF-8
-nl_NL@euro ISO-8859-15
-nn_NO ISO-8859-1
-nn_NO.UTF-8 UTF-8
-nr_ZA UTF-8
-nso_ZA UTF-8
-oc_FR ISO-8859-1
-oc_FR.UTF-8 UTF-8
-om_ET UTF-8
-om_KE ISO-8859-1
-om_KE.UTF-8 UTF-8
-or_IN UTF-8
-os_RU UTF-8
-pa_IN UTF-8
-pa_PK UTF-8
-pap_AN UTF-8
-pap_AW UTF-8
-pap_CW UTF-8
-pl_PL ISO-8859-2
-pl_PL.UTF-8 UTF-8
-ps_AF UTF-8
-pt_BR ISO-8859-1
-pt_BR.UTF-8 UTF-8
-pt_PT ISO-8859-1
-pt_PT.UTF-8 UTF-8
-pt_PT@euro ISO-8859-15
-quz_PE UTF-8
-raj_IN UTF-8
-ro_RO ISO-8859-2
-ro_RO.UTF-8 UTF-8
-ru_RU ISO-8859-5
-ru_RU.CP1251 CP1251
-ru_RU.KOI8-R KOI8-R
-ru_RU.UTF-8 UTF-8
-ru_UA KOI8-U
-ru_UA.UTF-8 UTF-8
-rw_RW UTF-8
-sa_IN UTF-8
-sat_IN UTF-8
-sc_IT UTF-8
-sd_IN UTF-8
-sd_IN@devanagari UTF-8
-sd_PK UTF-8
-se_NO UTF-8
-shs_CA UTF-8
-si_LK UTF-8
-sid_ET UTF-8
-sk_SK ISO-8859-2
-sk_SK.UTF-8 UTF-8
-sl_SI ISO-8859-2
-sl_SI.UTF-8 UTF-8
-so_DJ ISO-8859-1
-so_DJ.UTF-8 UTF-8
-so_ET UTF-8
-so_KE ISO-8859-1
-so_KE.UTF-8 UTF-8
-so_SO ISO-8859-1
-so_SO.UTF-8 UTF-8
-sq_AL ISO-8859-1
-sq_AL.UTF-8 UTF-8
-sq_MK UTF-8
-sr_ME UTF-8
-sr_RS UTF-8
-sr_RS@latin UTF-8
-ss_ZA UTF-8
-st_ZA ISO-8859-1
-st_ZA.UTF-8 UTF-8
-sv_FI ISO-8859-1
-sv_FI.UTF-8 UTF-8
-sv_FI@euro ISO-8859-15
-sv_SE ISO-8859-1
-sv_SE.ISO-8859-15 ISO-8859-15
-sv_SE.UTF-8 UTF-8
-sw_KE UTF-8
-sw_TZ UTF-8
-szl_PL UTF-8
-ta_IN UTF-8
-ta_LK UTF-8
-tcy_IN.UTF-8 UTF-8
-te_IN UTF-8
-tg_TJ KOI8-T
-tg_TJ.UTF-8 UTF-8
-th_TH TIS-620
-th_TH.UTF-8 UTF-8
-the_NP UTF-8
-ti_ER UTF-8
-ti_ET UTF-8
-tig_ER UTF-8
-tk_TM UTF-8
-tl_PH ISO-8859-1
-tl_PH.UTF-8 UTF-8
-tn_ZA UTF-8
-tr_CY ISO-8859-9
-tr_CY.UTF-8 UTF-8
-tr_TR ISO-8859-9
-tr_TR.UTF-8 UTF-8
-ts_ZA UTF-8
-tt_RU UTF-8
-tt_RU@iqtelif UTF-8
-ug_CN UTF-8
-ug_CN@latin UTF-8
-uk_UA KOI8-U
-uk_UA.UTF-8 UTF-8
-unm_US UTF-8
-ur_IN UTF-8
-ur_PK UTF-8
-uz_UZ ISO-8859-1
-uz_UZ.UTF-8 UTF-8
-uz_UZ@cyrillic UTF-8
-ve_ZA UTF-8
-vi_VN UTF-8
-wa_BE ISO-8859-1
-wa_BE.UTF-8 UTF-8
-wa_BE@euro ISO-8859-15
-wae_CH UTF-8
-wal_ET UTF-8
-wo_SN UTF-8
-xh_ZA ISO-8859-1
-xh_ZA.UTF-8 UTF-8
-yi_US CP1255
-yi_US.UTF-8 UTF-8
-yo_NG UTF-8
-yue_HK UTF-8
-zh_CN GB2312
-zh_CN.GB18030 GB18030
-zh_CN.GBK GBK
-zh_CN.UTF-8 UTF-8
-zh_HK BIG5-HKSCS
-zh_HK.UTF-8 UTF-8
-zh_SG GB2312
-zh_SG.GBK GBK
-zh_SG.UTF-8 UTF-8
-zh_TW BIG5
-zh_TW.EUC-TW EUC-TW
-zh_TW.UTF-8 UTF-8
-zu_ZA ISO-8859-1
-zu_ZA.UTF-8 UTF-8`
+type Locale struct {
```

Comment:
__[golint]__
* [golint] exported type Locale should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->